Makefile

```diff
@@ -97,17 +97,24 @@ install-worker-config:
 	sed 's,#LIBEXEC#,$(LIBEXECDIR),g' config/device-worker.toml > $(BUILDROOT)$(SYSCONFDIR)/yggdrasil/workers/device-worker.toml
 
 install: ## Install device-worker with debug enabled
-install-debug: build-debug install-worker-config
+install-debug: build-debug
+	sudo $(MAKE) install-worker-config
 	sudo install -D -m 755 ./bin/device-worker $(LIBEXECDIR)/yggdrasil/device-worker
 
 install: ## Install device-worker
-install: build install-worker-config
+install: build
+	sudo $(MAKE) install-worker-config
 	sudo install -D -m 755 ./bin/device-worker $(LIBEXECDIR)/yggdrasil/device-worker
 
 install-arm64: ## Install device-worker on arm64.
-install-arm64: build-arm64 install-worker-config
+install-arm64: build-arm64
+	sudo $(MAKE) install-worker-config
 	sudo install -D -m 755 ./bin/device-worker-aarch64 $(LIBEXECDIR)/yggdrasil/device-worker
 
+uninstall:
+	sudo rm -f $(SYSCONFDIR)/yggdrasil/device/device-config.json
+	sudo rm -rf /var/local/yggdrasil/device/manifests/*
```

Comment:
I believe we can remove the entire folder's content: `sudo rm -rf /var/local/yggdrasil/device/*`

Reply:
done